olive/evaluator/olive_evaluator.py

```diff
@@ -226,87 +230,46 @@ def generate_metric_user_config_with_model_io(metric: Metric, model: "OliveModel
                 "Model input shapes are not static. Cannot use inferred input shapes for creating dummy data. This will"
```

Comment:
this can be a direct failure since we know static shapes are needed to create dummy data for latency metric.

Reply:
This is to keep the old behavior intact. This is not a failure in current implementation.